### ISSUE TITLE
Standards for management

### ISSUE THREAD

**Issue by istfer** [PecanProject/pecan#2518]
## Description

As I add more and more crop models to PEcAn, it became an issue how to pass the management options to these models. Until now, I only dealt with passing harvest days to BASGRA and STICS for which I was using a simple text file format specifying cutting dates. BASGRA is a simple model, it only simulates cutting and fertilization for management but STICS has many many many options (fertilization composition, location of mineral nitrogen inputs, irrigation, location of irrigation, cutting height, mulch, thinning, pruning, sowing etc.). LPJ-GUESS agriculture simulations also have similar management settings. I suspect management in other models would also require a way of passing these options, e.g. https://github.com/PecanProject/pecan/issues/2122.

## Proposed Solution

Although there are differences from models to models, the logic is not too different (all use some decision to do the harvest, fertilization amount/composition which could be nested etc.). Overall,  I feel like we need a management workflow and/or a management standards file of sorts. 

I need to apply these management scenarios soon. Before inventing ways for handling my use cases, I thought it would be helpful to have some design discussion.

**Comment by ashiklom** [comment]
Also ED2 has anthropogenic disturbance. My understanding is that it takes an XML file that specifies PFTs, dates, and levels of disturbance.

**Comment by mdietze** [comment]
So Nell Campbell developed a couple small proposals (one to UNH, one to Brazil) about using PEcAn in an agronomic context, and part of that was to incorporate this sort of standardized management info. Looking quickly back over that text, it looks like this was going to be build off of things like http://farmos.org/   Nell is now at https://www.indigoag.com/ but this may still be very relevant to her, so I'd recommend reaching out

**Comment by dlebauer** [comment]
We should also look at how AgMIP encodes managements using the ICASA standard; @KristinaRiemer is working on exporting data from BETYdb to this format.

**Comment by istfer** [comment]
I'll drop this here for now https://docs.google.com/spreadsheets/d/1MYx1ukUsCAM1pcixbVQSu49NU-LfXg-Dtt-ncLBzGAM/pub?output=html#

**Comment by istfer** [comment]
@KristinaRiemer do you have a go-to resource for ICASA standards? Main main source is the google sheet I shared above, but I was wondering whether there is a more official/up-to-date place I should be looking

**Comment by dlebauer** [comment]
@istfer I am pretty sure that [those CSVs](https://github.com/agmip/VMapper/tree/develop/Data/ICASA) are dumped from the [google spreadsheet](https://docs.google.com/spreadsheets/d/1MYx1ukUsCAM1pcixbVQSu49NU-LfXg-Dtt-ncLBzGAM/pub?output=html#) @MengZhang can confirm?

It is also worth noting that that 'VMapper' repository is a tool to help map fields from a bespoke spreadsheet or json file to the ICASA variables.  The website is here http://vmapper-test.herokuapp.com/ and the Vmapper tool is here http://vmapper-test.herokuapp.com/tools/vmapper.

FYI @mengzhang @frostbytten and @chporter the PEcAn community was looking for standards for encoding managements / events etc so I pointed them to ICASA. Might be worth addressing at the next mini-hackathon (@istfer I'll contact you on slack w/ info).

**Comment by MengZhang** [comment]
@dlebauer You are correct. The CSV files you mentioned are the local cache dumped from ICASA google site for performance and version control purpose.

**Comment by dlebauer** [comment]
@istfer can this be closed since we have decided on a vocabulary (ICASA) and JSON Schema (https://github.com/hamk-uas/fieldobservatory-data-schemas/blob/main/management-event.schema.json)?

**Comment by dlebauer** [comment]
Closing this as complete - it has been discussed in #3551 and implemented in #3621. We can open new issues to iteratively improve the specification, starting with #3784
